admin_guide/pruning_resources.adoc

```diff
@@ -337,3 +390,37 @@ Such tags tend to have just one image in its history, which effectively prevents
 them from ever being pruned.
 xref:../dev_guide/managing_images.adoc#tag-naming[Learn more about _istag_
 naming.]
+
+[[using-secure-connection-against-insecure-registry]]
+==== Using secure connection against insecure registry
+
+If you see similar message in the output of the `oadm prune images` command:
+
+----
+error: error communicating with registry: Get https://172.30.30.30:5000/healthz: http: server gave HTTP response to HTTPS client
+----
+
+It means that your registry is not secured and the `oadm prune images` client
+attempts to use secure connection. The recommened solution is to
+xref:../install_config/registry/securing_and_exposing_registry.adoc#securing-the-registry[secure
+the registry]. If that is not desired, you may force the client to use insecure
+connection by appending `--force-insecure` to the command.
+
+[[using-insecure-connection-against-secured-registry]]
+==== Using insecure connection against secured registry
+
+If you see one of the following errors in the output of the `oadm prune images`
+command:
+
+----
+error: error communicating with registry: Get http://172.30.30.30:5000/healthz: malformed HTTP response "\x15\x03\x01\x00\x02\x02"
+error: error communicating with registry: [Get https://172.30.30.30:5000/healthz: x509: certificate signed by unknown authority, Get http://172.30.30.30:5000/healthz: malformed HTTP response "\x15\x03\x01\x00\x02\x02"]
+----
+
+It means that your registry is secured using a certificate signed by a
+certificate authority other than the one used by `oadm prune images` client for
+connection verification. By default, the certificate authority data stored in
```

Comment:
I would move the code block to just before "By default...." so that the first sentence does not get broken up.